xonsh/pyghooks.py

```diff
@@ -281,7 +282,8 @@ def __init__(self, style_name='default'):
         style_name : str, optional
             The style name to initialize with.
         """
-        self.trap = {}  # for custom colors
+        self.trap = {} # for traping custom colors set by user
+        self._mod = {} # for internal modification of the style
```

Comment:
Is there a problem with just using trap? This was exactly the use case for it.  I don't think there is a need to two traps...